### ISSUE TITLE
how to connect multiple sub tunnel in one port.

### ISSUE THREAD

**Issue by rishabhsingh2** [markfoodyburton/ubond#5]
server side : 
[l1r1]
bindhost = "192.168.aa.aa"
bindport = 32108

[l1r2]
bindhost = "192.168.bb.bb"
bindport =  32108

[l2r1]
bindhost = "192.168.aa.aa"
bindport = 32108

[l2r2]
bindhost = "192.168.bb.bb"
bindport = 32108

client side:
[l1r1]
bindhost = "192.168.xx.xx"
bindport = 5080
remotehost = "192.168.aa.aa"
remoteport = 32108
[l1r2]
bindhost = "192.168.xx.xx"
bindport = 5081
remotehost = "192.168.bb.bb"
remoteport = 32108
[l2r1]
bindhost = "192.168.yy.yy"
bindport = 5082
remotehost = "192.168.aa.aa"
remoteport = 32108
[l2r2]
bindhost = "192.168.yy.yy"
bindport = 5083
remotehost = "192.168.bb.bb"
remoteport = 32108

i have write this type in both side but sub tunnel is not created.
how to connect ?

**Comment by markfoodyburton** [comment]
in the server side, you need to bind the host and port. On the client side you need the remote host and port (and they must all match), you should not have 'bind host/port' on the client side. You will probably need an up/down script too.

**Comment by rishabhsingh2** [comment]
yes , server side i have bind the host and port . and what to do on client side ?
in client side which host and port are bind? 
what make changes in up/down script ?

**Comment by markfoodyburton** [comment]
you should have remotehost/port (ONLY) on client side.
The updown script sets up the device etc - there is one installed by default, mostly it should work for you. You mayweed to sort out routing etc. Good luck.

**Comment by rishabhsingh2** [comment]
yes ,  client side only give remotehost/port but server side not up all sub tunnels.

**Comment by markfoodyburton** [comment]
is your problem with the tunnel or the device?

**Comment by rishabhsingh2** [comment]
yes problem in sub tunnels.

**Comment by markfoodyburton** [comment]
use verbose to see what's happening.
